getambassadorio_blc.py

```diff
@@ -225,18 +226,16 @@ def _parse_srcset_value(attrvalue: str) -> List:
         return [desc.split()[0] for desc in attrvalue.split(',')]
 
 
-def main(
-    checkerCls: CheckerInterface, projdir: str, pages_to_check_file: str, base_address: str
-) -> int:
+def main(checkerCls: CheckerInterface, projdir: str, pages_to_check_file: str) -> int:
     urls = [
-        f'{base_address}/',
-        f'{base_address}/404.html',
-        f'{base_address}/404/',
+        'http://localhost:9000/',
+        'http://localhost:9000/404.html',
+        'http://localhost:9000/404/',
```

Comment:
Did you mean to change this back to `localhost:9000`? Kind of seems like this accidentally got reverted